README.md

```diff
@@ -217,6 +217,7 @@ Below is a list of known projects that use Badger:
 * [flow-go](https://github.com/onflow/flow-go) - A fast, secure, and developer-friendly blockchain built to support the next generation of games, apps and the digital assets that power them.
 * [Wrgl](https://www.wrgl.co) - A data version control system that works like Git but specialized to store and diff CSV.
 * [Loggie](https://github.com/loggie-io/loggie) - A lightweight, cloud-native data transfer agent and aggregator.
+* [raft-badger](https://github.com/rfyiamcool/raft-badger) - raft-badger implements LogStore and StableStore Interface of hashcorp/raft. it is used to store raft log and metadata of hashcorp/raft.
```

Comment:
can you also update the file docs/content/projects-using-badger/index.md. I just realized that these two are not in sync, so created a PR https://github.com/dgraph-io/badger/pull/1903/files.